src/wwwroot/services/templates.js

```diff
@@ -69,14 +83,34 @@
         request.actionDescription = 'action_templates_creating',
         request.method = 'POST',
         request.url = RELAY_CONFIG.baseUrl + '/accounts/' + accountId + '/templates';
+
+        return $http(request).then(function (result) {
+          return result.createdResourceId;
+        });
       } else {
+        var body = request.data.body;
+        // Prevent to update body using deprecated property
+        request.data.body = "";
         request.actionDescription = '"Editing a template"',
         request.method = 'PUT',
         request.url = RELAY_CONFIG.baseUrl + '/accounts/' + accountId + '/templates/' + model.id;
+        return editTemplateBody(accountId, model.id, request.data.name, body).then(function (r) {
```

Comment:
Did you think how to deal with errors? For example, what happen if one request works and the another fails?

Reply:
I will investigate it, thanks

Reply:
What do you think about to create a template model backup to rollback if the second request fails?